### PR TITLE
Add Getting Started Page for Users

### PR DESCRIPTION
**Pull request by Varunram**
Intended as a fix for #1673.
Didn't know where to add the file, so put it under `docs/usage/`
Thanks!

### CODE REVIEW

File: docs/docs/usage/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+# Run it with the proper classpath
+$ dotr HelloWorld
+```
+
+
+Starting a REPL
+----------------
+```bash
+$ sbt
+> repl
```
Comment (nicolasstucki) [inline docs/docs/usage/getting-started.md]: This will only work on a project that has a dotty enabled for the repl such as `lampepfl/dotty.g8`.

**Comment by felixmulder** [comment]
@biboudis: sounds good to me :+1:

**Comment by Varunram** [comment]
Awesome! Will do the changes right away. Sorry for being late!

**Comment by Varunram** [comment]
@biboudis Changes done as requested. Thanks for your time!

**Comment by biboudis** [comment]
Perfect. I think the file `docs/docs/usage/getting-started.md` is not needed at all.

**Comment by Varunram** [comment]
@biboudis Yeah, that's what I was thinking too. But since @felixmulder raised an issue I put in a new page. Waiting for Felix's review

**Comment by biboudis** [comment]
@Varunram let's add the new information that is documented on http://dotty.epfl.ch/#getting-started 

Namely `brew install lampepfl/brew/dotty` for Mac and getting the latest release for the rest. Put that before the *Getting Dotty* section, titled *Install Dotty*.

Also, don't forget to add the necessary entry in https://github.com/lampepfl/dotty/blob/master/docs/sidebar.yml 
to create the new navigation option.

**Comment by Varunram** [comment]
@biboudis Done!

File: docs/docs/usage/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+
+
+Basics
+------------
+Make sure that you are using Java 8 or later. The output of `java -version`
+should contain `1.8`.
+
+
+Install Dotty
```
Comment (biboudis) [inline docs/docs/usage/getting-started.md]: Can you make it "Option 1: Install Dotty"?

File: docs/docs/usage/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+-------------
+If you're a Mac user, you can install dotty with [brew](https://brew.sh/)
+
+```bash
+brew install lampepfl/brew/dotty
+```
+
+If you're a Linux or Windows user, download the [latest release](https://github.com/lampepfl/dotty/releases). Optionally add path of the folder `bin/` to the system environment variable `PATH`.
+
+Getting Dotty
```
Comment (biboudis) [inline docs/docs/usage/getting-started.md]: `Option 2: Using Dotty directly from source`

File: docs/docs/usage/getting-started.md (hunk 2 above)
Comment (Varunram) [inline docs/docs/usage/getting-started.md]: Sure, will do :)

**Comment by Varunram** [comment]
Thanks a ton for your patience! Really appreciate it :)

**Pull request merged**